roles/eth2client/defaults/teku/beacon.yml

```diff
@@ -75,5 +75,5 @@ secrets_container_dir: "{{beacon_container_dir}}/secrets"
 keystores_host_dir: "{{beacon_host_dir}}/keys"
 secrets_host_dir: "{{beacon_host_dir}}/secrets"
 
-beacon_keystores_file_mode: "0700"  # execute bit, to iterate directory
+beacon_keystores_file_mode: "0704"  # execute bit, to iterate directory
```

Comment:
why change the permissions here?

Reply:
Teku failed for me stating it cannot read the keystores, setting the "other" to read access with "704" fixed the issue. I'm not sure what changed, maybe its to do with how mac generates and stores keys vs linux? They should be the same, but that's the only change I can know of between our systems. 

I could change the variable in my directory instead of changing the default value though. Maybe that's the more secure option.

Reply:
Due to keystore permissions or lockfiles? Maybe they changed the docker user again, or there's some other change. Can you ping them on discord or open an issue on the teku repo? They probably know what it should be.

Reply:
It was due to keystore permissions. I can test it out again, collect logs and open an issue on their repo.